runtime/westend/Cargo.toml

```diff
@@ -0,0 +1,135 @@
+[package]
+name = "westend-runtime"
```

Comment:
Why does it have it's own runtime?

Reply:
So the initial idea was to  copy the runtime folder from kusama since this network is supposed to act like kusama in general and have a place to store these changes. Some of the polkadot defaults do not much the kusama specs.

One for example is ```EPOCH_DURATION_IN_BLOCKS``` which should be matched with kusama. Is there another place where it's better to put this like the ```chain_spec.rs``` with some changes?

Reply:
How did you build the westend chainspec? The problem is that the runtime is not correctly integrated into the project, so I think that you probably have generated an incorrect genesis.

Reply:
I had actually added this later and wanted to use it for a test of a runtime upgrade for the EPOCH_DURATION_IN_BLOCKS. But I can just remove this for now

Reply:
Reverted.